sql/synth_recipes.sql

```diff
@@ -3597,25 +3597,25 @@ INSERT INTO `synth_recipes` VALUES (59928,1,0,0,0,0,0,0,7,0,0,4100,4242,13442,0,
 INSERT INTO `synth_recipes` VALUES (59930,1,0,0,0,0,0,0,28,0,0,4100,4242,12455,0,0,0,0,0,0,0,852,852,894,894,1,1,1,1,'Beetle Mask (desynth)');
 INSERT INTO `synth_recipes` VALUES (59931,1,0,0,0,0,0,0,92,0,0,4100,4242,14305,0,0,0,0,0,0,0,1629,1629,1629,1629,1,1,1,1,'Dragon Subligar (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59932,1,0,0,0,0,0,0,255,0,0,4100,4242,14852,0,0,0,0,0,0,0,1163,1622,790,1618,1,1,1,1,'Igqira Manillas (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59933,1,0,0,0,0,0,0,255,0,0,4100,4242,12878,0,0,0,0,0,0,0,819,506,887,887,2,1,1,1,'Coral Subligar (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59934,1,0,0,0,0,0,0,255,0,0,4100,4242,13713,0,0,0,0,0,0,0,864,864,864,864,1,1,1,1,'Carapace Mittens (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59935,1,0,0,0,0,0,0,255,0,0,4100,4242,17026,0,0,0,0,0,0,0,880,880,880,880,2,2,2,2,'Bone Cudgel (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59936,1,0,0,0,0,0,0,255,0,0,4100,4242,13850,0,0,0,0,0,0,0,887,887,887,887,1,1,1,1,'Merman\'s Hairpin (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59937,1,0,0,0,0,0,0,255,0,0,4100,4242,12583,0,0,0,0,0,0,0,852,889,889,889,3,1,1,1,'Beetle Harness (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59938,1,0,0,0,0,0,0,255,0,0,4100,4242,13846,0,0,0,0,0,0,0,850,1230,1473,1473,1,6,1,1,'Scorpion Helm (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59939,1,0,0,0,0,0,0,255,0,0,4100,4242,14008,0,0,0,0,0,0,0,817,1193,850,850,3,1,3,3,'Cpc. Gauntlets (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59940,1,0,0,0,0,0,0,255,0,0,4100,4242,17849,0,0,0,0,0,0,0,2157,2171,2171,2171,1,1,1,1,'Hellish Bugle (desynth)');
+INSERT INTO `synth_recipes` VALUES (59933,1,0,0,0,0,0,0,72,0,0,4100,4242,12878,0,0,0,0,0,0,0,819,506,887,887,2,1,1,1,'Coral Subligar (desynth)');
+INSERT INTO `synth_recipes` VALUES (59934,1,0,0,0,0,0,0,44,0,0,4100,4242,13713,0,0,0,0,0,0,0,864,864,864,864,1,1,1,1,'Carapace Mittens (desynth)');  -- 2021.11.01
```

Comment:
its ok to use synth lv for the desynth, but we need to state it in the comment.

Reply:
I will update accordingly.